website/source/docs/builders/azure.html.md

```diff
@@ -53,9 +61,10 @@ builder.
 
     CLI example `azure vm image list-skus -l westus -p Canonical -o UbuntuServer`
 
--   `location` (string) Azure datacenter in which your VM will build.
+##### or:
```

Comment:
I would strengthen this more to ensure it's clear it's and either other proposition.